charts/k8s-monitoring/values.yaml

```diff
@@ -2029,6 +2080,18 @@ opencost:
     nodeSelector:
       kubernetes.io/os: linux
 
+# Settings for the Kepler deployment
+# You can use this sections to make modifications to the Kepler deployment.
+# See https://github.com/sustainable-computing-io/kepler-helm-chart/tree/main/chart/kepler for available values.
+kepler:
+  # -- Should this Helm chart deploy Kepler to the cluster.
+  # Set this to false if your cluster already has Kepler, or if you do not want to scrape metrics from Kepler.
+  # @section -- Chart
+  enabled: false
+
+  # @ignored -- This skips including these values in README.md
+  canMount:
+    usrSrc: false
```

Comment:
`values.yaml` does have a `nodeSelector` field defaulting to
```
nodeSelector:
  kubernetes.io/os: linux
```
, suggest adding also amd64 constraint, as per https://kubernetes.io/docs/reference/labels-annotations-taints/#kubernetes-io-arch
```
  kubernetes.io/arch: amd64
```

Reply:
Yeah, I really thought about adding this. My hesitation is that there are ARM platforms that are supported, and I don't want to cut them out.

Is it easier to have the amd64 restriction and have to explain how to remove if you have a supported arm node, or don't have the restriction and have a doc about why pods are crashing on graviton nodes. I'm leaning towards the latter, but happy to discuss.

Reply:
I looked a long time to determine if I can detect if the pod is on a graviton node, but I don't think it's possible at pod scheduling time.